python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -868,7 +868,7 @@ def countMaskFromFootprint(mask, footprint, bitmask, ignoreMask):
     bbox.clip(mask.getBBox(afwImage.PARENT))
     fp = afwImage.MaskU(bbox)
     subMask = mask.Factory(mask, bbox, afwImage.PARENT)
-    afwDet.setMaskFromFootprint(fp, footprint, bitmask)
+    footprint.spans.setMask(fp, bitmask)
```

Comment:
Huge 👍 on the improved (and more pythonic) API here! I especially like that you can just call ``footprint.spans`` as opposed to ``footprint.getSpans()``.